examples/telemetry.py

```diff
@@ -15,7 +15,7 @@ async def run():
     await drone.connect(system_address="udpin://0.0.0.0:14540")
 
     # Start the tasks
-    tasks = [
+    _tasks = [
```

Comment:
It is unusual to assemble these tasks but not await them.  Are we certain that they will be run at some point?